textinput/textinput.go

```diff
@@ -78,8 +80,11 @@ var DefaultKeyMap = KeyMap{
 	LineEnd:                 key.NewBinding(key.WithKeys("end", "ctrl+e")),
 	Paste:                   key.NewBinding(key.WithKeys("ctrl+v")),
 	AcceptSuggestion:        key.NewBinding(key.WithKeys("tab")),
-	NextSuggestion:          key.NewBinding(key.WithKeys("down", "ctrl+n")),
-	PrevSuggestion:          key.NewBinding(key.WithKeys("up", "ctrl+p")),
+	NextSuggestion:          key.NewBinding(key.WithKeys("ctrl+n")),
+	PrevSuggestion:          key.NewBinding(key.WithKeys("ctrl+p")),
+	// TODO: discuss moving `down` and `up` to this instead of Next/PrevSuggestion
+	NextHistory: key.NewBinding(key.WithKeys("down")),
+	PrevHistory: key.NewBinding(key.WithKeys("up")),
```

Comment:
Up and Down feel more natural to me, as a shell/vim user than updating completion. However this might break existing applications and change the default behavior, so I'd be happy not to change it.

Which keys would you recommend?